This is a checksum fix for building LAMMPS-29Sep2021 documentation and cuda-aware problem
--- lammps-stable_29Sep2021/cmake/Modules/Documentation.cmake.orig	2021-12-03 16:35:31.125375000 +0000
+++ lammps-stable_29Sep2021/cmake/Modules/Documentation.cmake	2021-12-03 16:35:31.141434000 +0000
@@ -56,7 +56,7 @@
   )
 
   set(MATHJAX_URL "https://github.com/mathjax/MathJax/archive/3.1.3.tar.gz" CACHE STRING "URL for MathJax tarball")
-  set(MATHJAX_MD5 "d1c98c746888bfd52ca8ebc10704f92f" CACHE STRING "MD5 checksum of MathJax tarball")
+  set(MATHJAX_MD5 "b81661c6e6ba06278e6ae37b30b0c492" CACHE STRING "MD5 checksum of MathJax tarball")
   mark_as_advanced(MATHJAX_URL)
 
   # download mathjax distribution and unpack to folder "mathjax"
--- lammps-stable_29Sep2021/src/KOKKOS/kokkos_old.cpp	2021-12-03 16:30:57.997173000 +0000
+++ lammps-stable_29Sep2021/src/KOKKOS/kokkos.cpp	2021-12-03 16:33:20.171097593 +0000
@@ -276,9 +276,9 @@
 
     if (gpu_aware_flag == 1 && have_gpu_aware == 0) {
       if (me == 0)
-        error->warning(FLERR,"Turning off GPU-aware MPI since it is not detected, "
-                       "use '-pk kokkos gpu/aware on' to override");
-      gpu_aware_flag = 0;
+        error->warning(FLERR,"GPU-aware MPI is not detected, "
+                       "use '-pk kokkos gpu/aware off'  in case of segfaults");
+      //gpu_aware_flag = 0;
     } else if (have_gpu_aware == -1) { // maybe we are dealing with MPICH, MVAPICH2 or some derivative?
     // MVAPICH2
 #if defined(MPICH) && defined(MVAPICH2_VERSION)
